src/spacel/provision/alarm/endpoint/pagerduty.py

```diff
@@ -167,7 +167,7 @@ def _pd_api(self, url, data=None, method='GET'):
         request_args = {
             'headers': dict(self._pd_headers)
         }
-        if six.PY3:
+        if six.PY3: # pragma: no cover
```

Comment:
Autoformat agrees: it wa supposed to be a ninja-patch after looking at coverage diffs between py2/3.

Fixed.